dangerous.php

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <?php require get_template_directory() . '/inc/template-head-generic.php'; ?>
+    <script src="//cdn1.readspeaker.com/script/8419/webReader/webReader.js?pids=wr" type="text/javascript"></script>
```

Comment:
If 8419 is a customer number or something like that, it should be in the settings

Reply:
Fixed